Week 2/Photo_Gallary.css

```diff
@@ -1,3 +1,3 @@
-img {
+img{
```

Comment:
Not a big deal, but best practices say add a space after the CSS selector and use two spaces for indentation. This goes for JS as well.

```css
img {
  vertical-align: text-bottom;
}
```